src/maintenance/repositories/official/multilib.md

```diff
@@ -0,0 +1,9 @@
+# multilib
+
+The `multilib` repository provides 32-bit packages as a compatibility layer
+inside a 64-bit system and is available through the package
+`void-repo-multilib`. The repository also has a sub-repository containing
```

Comment:
again, language change for sub-

Reply:
My thinking was that the non-free multilib repository depends on the multilib repository.